src/common/map/MapService.js

```diff
@@ -721,6 +721,22 @@
           } else {
             console.log('====[ Error: could not create base layer.');
           }
+        } else if (server.ptype === 'gxp_arcrestsource') {
+          var attribution = new ol.Attribution({
+            html: 'Tiles &copy; <a href="http://services.arcgisonline.com/ArcGIS/rest/services/World_Topo_Map/MapServer">ArcGIS</a>'
+          });
+          layer = new ol.layer.Tile({
+            metadata: {
+              serverId: server.id,
+              name: minimalConfig.name,
+              title: fullConfig.Title
+            },
+            visible: minimalConfig.visibility,
+            source: new ol.source.XYZ({
+              attributions: [attribution],
+              url: 'http://server.arcgisonline.com/ArcGIS/rest/services/World_Topo_Map/MapServer/tile/{z}/{y}/{x}'
```

Comment:
Does it make sense to get the url from `server.url` so that different endpoint can be configured?
This seems to work for gxp
 {
         "source": {"ptype": "gxp_olsource", "name": "ESRI"},
         "args":["World Map","http://z_y_x",{"layers":'basic'}],
         "visibility":True,
         "fixed":False,
         "group":"background",
         "type": "OpenLayers.Layer.XYZ"
}